### ISSUE TITLE
Truely random?

### ISSUE THREAD

**Issue by amrrs** [amrrs/randquotes#9]
Currently because we're looking at the first page, every time in a given day it seems like it's the same Quote.

https://quotesondesign.com/wp-json/wp/v2/posts/?orderby=rand&per_page=1

Can we really mke it random? Should we?

**Comment by KKulma** [comment]
Yes, I noticed that when developing the code but this is the recommended way of calling random quotes according to the [API page](https://quotesondesign.com/api/). IMHO, We should contact the API maintainer about it.
